pkgs/tools/filesystems/btrfs-progs/default.nix

```diff
@@ -21,6 +21,11 @@ stdenv.mkDerivation rec {
   # This should be fine on all platforms so apply universally
   patchPhase = "sed -i s/-O1/-O2/ configure";
 
+  postInstall = ''
+    install -vd $out/etc/bash_completion.d
+    install -vm 644 btrfs-completion $out/etc/bash_completion.d/btrfs
```

Comment:
Thanks, change as suggested (25028cb)

Can you explain why `444` mode should be using instead of `644`?